lib/tasks/message_queue.rake

```diff
@@ -0,0 +1,21 @@
+namespace :message_queue do
+  desc "Index documents that are published to the publishing-api"
+  task :index_documents_from_publishing_api do
+    # The following environment variables need to be set
+    # RABBITMQ_HOSTS
+    # RABBITMQ_PORT
```

Comment:
RABBITMQ_PORT doesn't appear to be a thing that exists in the message consumer code.  I think we're just using the default AMQP port always.